docs/manual/manual.adoc

```diff
@@ -233,6 +233,16 @@ Definition file, *usgcb-rhel5desktop-oval.xml-0.variables-0.xml* is the
 file containing exported variables from the XCCDF file, and
 *usgcb-results-oval.xml* is the the OVAL Result file.
 
+An OVAL directives file can be used to control whether results should be "thin" or "full".
+This file can be loaded by OpenSCAP using *--directives <file>* option.
+
+OVAL results file contains, by default, exported system characteristics.
+OpenSCAP provides *--without-syschar* option to change this behavior.
```

Comment:
Sure, https://github.com/OpenSCAP/openscap/pull/518#issuecomment-246272257 
I didn't want to separate this changes during reviewing.

Should I remove --without-syschar, now?